src/Documentation/SidebarMenu/SidebarMenu.js

```diff
@@ -9,6 +10,24 @@ import styled from 'styled-components'
 import { media, OnlyDesktop } from '../../styles'
 
 export default class SidebarMenu extends React.Component {
+    constructor(props) {
+        super(props);
+        this.collapse = this.collapse.bind(this);
+    }
+    collapse(){
+        setTimeout(function () {
+            $('[data-open=true]').slideDown();
```

Comment:
please, let's use 2 spaces as in other code

Reply:
Ok, there were 4 spaces in the source code

Reply:
Yep. Looks there was a mix. Let's change it please, so that we have it consistent. Thanks!